jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/DrowdownTests.java

```diff
@@ -0,0 +1,257 @@
+        largeSplitDropdown.dropdownToggle.is()
+                .core()
+                .cssClass(containsString("btn-lg"))
+                .cssClass(containsString("dropdown-toggle-split"))
+                .css("padding", is("8px 12px"))
+                .css("font-size", is("20px"))
+                .css("line-height", is("30px"))
+                .css("border-radius", is("0px 4.8px 4.8px 0px"));
+    }
+
+    @Test
+    public void smallDropdownIsValidationTest() {
+        smallDropdown.dropdownToggle.is()
+                .core()
+                .cssClass(containsString("btn-sm"))
+                .css("padding", is("4px 8px"))
+                .css("font-size", is("14px"))
+                .css("line-height", is("21px"))
+                .css("border-radius", is("3.2px"));
+    }
+
+    @Test
+    public void smallSplitDropdownIsValidationTest() {
+        smallSplitDropdown.dropdownToggle.is()
+                .core()
+                .cssClass(containsString("btn-sm"))
+                .cssClass(containsString("dropdown-toggle-split"))
+                .css("padding", is("4px 6px"))
+                .css("font-size", is("14px"))
+                .css("line-height", is("21px"))
+                .css("border-radius", is("0px 3.2px 3.2px 0px"));
+    }
+
+    @Test
+    public void dropUpDropdownIsValidationTest() {
+        dropUpDropdown.is()
+                .core()
+                .cssClass(containsString("dropup"));
+        dropUpDropdown.dropdownMenu.is()
+                .core()
+                .css("top", is("auto"))
+                .css("bottom", is("100%"));
+    }
+
+    @Test
+    public void dropUpSplitDropdownIsValidationTest() {
+        dropUpSplitDropdown.is()
+                .core()
+                .cssClass(containsString("dropup"));
+        dropUpSplitDropdown.dropdownMenu.is()
+                .core()
+                .css("top", is("auto"))
+                .css("bottom", is("100%"));
+    }
+
+    @Test
+    public void dropRightDropdownIsValidationTest() {
+        dropRightDropdown.is()
+                .core()
+                .cssClass(containsString("dropright"));
+        dropRightDropdown.dropdownMenu.is()
+                .core()
+                .css("right", is("auto"))
+                .css("left", is("100%"));
+    }
+
+    @Test
+    public void dropRightSplitDropdownIsValidationTest() {
+        dropRightSplitDropdown.is()
+                .core()
+                .cssClass(containsString("dropright"));
+        dropRightSplitDropdown.dropdownMenu.is()
+                .core()
+                .css("right", is("auto"))
+                .css("left", is("100%"));
+    }
+
+    @Test
+    public void dropLeftDropdownIsValidationTest() {
+        dropLeftDropdown.is()
+                .core()
+                .cssClass(containsString("dropleft"));
+        dropLeftDropdown.dropdownMenu.is()
+                .core()
+                .css("left", is("auto"))
```

Comment:
please create constant variables for left and right class names to not repeat it again and again as raw text